pkg/testcase/upgradenodereplacement.go

```diff
@@ -287,7 +292,8 @@ func joinRemainServers(
 	oldServerIPs []string,
 	serverLeaderIp,
 	token,
-	version string,
+	version,
+	channel string,
```

Comment:
same. add to line above with version

Reply:
I think its better keep the vars in separate line